ts/commands/ElementCommands.ts

```diff
@@ -63,6 +63,42 @@ class ElementCommands {
         const config = configService.mergeConfig(options);
         elementService.stopMaintenanceMode(args.id, config, logger);
       });
+
+    (caporal as any)
+      .command('element customtype image rm', 'Delete a custom image')
```

Comment:
This is a long command. What do you think about `image custom delete` or something like that?